### PR TITLE
User-story: 12829

### PR DESCRIPTION
**Pull request by ws-mps**
Stock Move Line report
New view for stock.move.lines with some predefined filters and groups.

### CODE REVIEW

**Review by apatrick88 — APPROVED** [review]

File: addons/udes_stock/views/stock_op_analysis.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    <record id="udes_stock.stock_op_analysis_tree" model="ir.ui.view">
+      <field name="name">udes_stock.stock.move.line.tree</field>
+      <field name="model">stock.move.line</field>
+      <field name="arch" type="xml">
+
+        <tree string="Move Line" create="0" decoration-muted="state == 'cancel'" decoration-danger="(state not in ('cancel','done'))">
+      	  <field name="id" invisible="1"/>
+          <field name="location_id"/>
+          <field name="u_done_by" />
+          <field name="u_done_datetime"/>
```
Comment (kenvac) [inline addons/udes_stock/views/stock_op_analysis.xml:14]: This field comes from module udes_scanned_by_tracking and udes_stock is not dependent on that module. 
If someone uses the udes_stock module wouldn't this report throw error?

File: addons/udes_stock/views/stock_op_analysis.xml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    <!-- Move Line Report tree view -->
+    <record id="udes_stock.stock_op_analysis_tree" model="ir.ui.view">
+      <field name="name">udes_stock.stock.move.line.tree</field>
+      <field name="model">stock.move.line</field>
+      <field name="arch" type="xml">
+
+        <tree string="Move Line" create="0" decoration-muted="state == 'cancel'" decoration-danger="(state not in ('cancel','done'))">
+      	  <field name="id" invisible="1"/>
+          <field name="location_id"/>
+          <field name="u_done_by" />
```
Comment (kenvac) [inline addons/udes_stock/views/stock_op_analysis.xml:13]: This field comes from module udes_scanned_by_tracking. See my comment below

**Review by kenvac — COMMENTED** [review]
Left a comment

File: addons/udes_stock/views/stock_op_analysis.xml (hunk 1 above)
Comment (ws-mps) [reply addons/udes_stock/views/stock_op_analysis.xml:14]: Thanks, added dependency.

File: addons/udes_stock/views/stock_op_analysis.xml (hunk 2 above)
Comment (ws-mps) [reply addons/udes_stock/views/stock_op_analysis.xml:13]: Thanks, added dependency.

**Review by kenvac — APPROVED** [review]

**Pull request merged**